### PR TITLE
✨ Add badge to bento component docs

### LINKED ISSUE #5094: Render Bento label in component reference docs
Once `bento` has been add to the supported formats in bento component readme files (see https://github.com/ampproject/amphtml/issues/31609), we should add a bento label to the respective component doc pages. 

* For the design we should re-use the `experimental` label design (color `#f15a5a`).  
* The label should link to the Bento intro guide (to be written).

Mockup: 

Screen Shot 2020-12-15 at 19 27 27

### PR DESCRIPTION
**Pull request by matthiasrohmer**
Resolves #5094.

Bildschirmfoto 2021-01-09 um 16 58 48

### CODE REVIEW

File: frontend/templates/views/partials/component-headline.j2 [hunk 1]
```diff
@@ -13,15 +13,26 @@
 {% endif %}
 </h1>
 
-{% if doc.experimental or doc.deprecated %}
+{% if doc.experimental or doc.deprecated or doc.bento %}
```
Comment (sebastianbenz) [inline frontend/templates/views/partials/component-headline.j2:16]: Would it make sense to add bento to the `formats` field?

**Review by sebastianbenz — APPROVED** [review]
Thanks Matthias!

File: frontend/templates/views/partials/component-headline.j2 (hunk 1 above)
Comment (matthiasrohmer) [reply frontend/templates/views/partials/component-headline.j2:16]: Could make sense. Is there a specific plan how bento component docs will be structured? Or do they have API parity with the classic components and therefore don't need their own docs but really just a marker?

Wasn't comfortable making assumptions about this and therefore simply added the flag 😁

Comment (sebastianbenz) [reply frontend/templates/views/partials/component-headline.j2:16]: Bento and AMP component docs are going to be the same (Bento and AMP version of a component is the same JS script). So a flag is fine.

**Review by sebastianbenz — APPROVED** [review]

**Pull request merged**